config/config.go

```diff
@@ -170,9 +180,9 @@ type ProxyConfig struct {
 	ComponentConfig
 	Endpoint string   `json:"endpoint" required:"true"`
 	Timeout  Duration `json:"timeout"`
-	Protocol string   `json:"protocol"`
-	Service  string   `json:"service"`
-	Method   string   `json:"method"`
+	Protocol string   `json:"protocol,omitempty"`
+	Service  string   `json:"service,omitempty"`
```

Comment:
Shall we move these into a `GrpcConfig` struct? Everything except `Service` and `Method` are applicable to both Http and grpc, right?

Reply:
`Service` and `Method` are only exclusive to grpc too. 

For http, the `url` and `method` are set within `http.Request` of the `Request` payload. For grpc, this information is stored in dispatcher.